contracts/staking/stakeManager/StakeManagerExtension.sol

```diff
@@ -128,13 +128,18 @@ contract StakeManagerExtension is StakeManagerStorage, Initializable, StakeManag
     function migrateValidatorsData(uint256 validatorIdFrom, uint256 validatorIdTo) external {       
         for (uint256 i = validatorIdFrom; i < validatorIdTo; ++i) {
             ValidatorShare contractAddress = ValidatorShare(validators[i].contractAddress);
-            if (contractAddress != ValidatorShare(0)) {
-                validators[i].delegatorsReward = contractAddress.validatorRewards_deprecated().add(INITIALIZED_AMOUNT);
+            if (contractAddress != ValidatorShare(0) && !validators[i].migrated) {
+                validators[i].migrated = true;
```

Comment:
Do we need it because we will be running migration only once?

Reply:
Not necessary but was thinking to prevent any potential breaking